### ISSUE TITLE
Update to work with EE 3.0+

### ISSUE THREAD

**Issue by bunnyhugdev** [rsanchez/eecli#24]
I'm not sure what all would need to be updated to make this work with the v3 release of EE. Running the tool in the base directory of a brand new EE3 project states that it cannot find the system folder.

**Comment by rsanchez** [comment]
Yes, this only works with EE2. I should make this clear in the README.

I have no plans to update this to work with EE3. There's a lot of work to update it. This could change in the future, but that's my stance for now.
